src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,142 @@
+package nextstep.subway.line;
+
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.line.dto.LinesResponse;
+import nextstep.subway.line.dto.ModifyLineRequest;
+import nextstep.subway.station.dto.StationResponse;
+import nextstep.subway.utils.DatabaseCleanup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.List;
+
+import static nextstep.subway.util.LineStep.*;
+import static nextstep.subway.util.StationStep.여러_개의_지하철_역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles("test")
+public class LineAcceptanceTest {
+
+    @Autowired
+    DatabaseCleanup databaseCleanup;
+
+    @BeforeEach
+    public void setup() {
+
+        databaseCleanup.execute();
+
+    }
+
+    /* Given: 새로운 지하철 노선 정보를 입력하고,
+       When: 관리자가 노선을 생성하면,
+       Then: 해당 노선이 생성되고 노선 목록에 포함된다. */
+    @DisplayName("지하철 노선을 생성한다")
+    @Test
+    public void createLine() {
+        // given
+        List<StationResponse> 지하철_역_목록 = 여러_개의_지하철_역_생성(List.of("강남역", "역삼역"));
+
+        // when
+        LineResponse 원본_지하철_노선 = new LineResponse("신분당선", "Red", List.of(지하철_역_목록.get(0), 지하철_역_목록.get(1)), 10L);
+        LineResponse 생성된_지하철_노선 = 지하철_노선_생성("신분당선", "Red", 지하철_역_목록.get(0).getId(), 지하철_역_목록.get(1).getId(), 10L);
```

Comment:
```suggestion
        var 지하철_역_목록 = 여러_개의_지하철_역_생성(List.of("강남역", "역삼역"));

        // when
       var 원본_지하철_노선 = new LineResponse("신분당선", "Red", List.of(지하철_역_목록.get(0), 지하철_역_목록.get(1)), 10L);
        var 생성된_지하철_노선 = 지하철_노선_생성("신분당선", "Red", 지하철_역_목록.get(0).getId(), 지하철_역_목록.get(1).getId(), 10L);
```
타입 추론을 사용해보는 건 어떨까요? 저는 타입 추론을 통하여 가독성이 상승하고, 좀 더 의미있는 변수 명을 지을 수 있다고 생각합니다.